src/cloudlabs/forms/customise_setup.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms.fields import TextAreaField, HiddenField
+import wtforms.validators as v
+
+
+class CustomiseSetupForm(FlaskForm):
+    id = HiddenField()
+    setup_script = TextAreaField('Setup script',
+                                 description='Specify the (bash) script run \
```

Comment:
I don't like using backslash to escape newlines, and this will also add lots of spaces to the string because of the indent. Better would be
```
description='Specify the (bash) script run'
                     ' to initialise the new host',
```
using Python's string joining.

(The string starting quotes should line up; I'm not sure why the Markdown is breaking that!)